libtcmu.c

```diff
@@ -383,18 +383,192 @@ static void tcmu_cdb_debug_info(const struct tcmulib_cmd *cmd)
                 free(buf);
 }
 
-static void *tcmu_cmdproc_thread(void *arg)
+static void _cleanup_mutex_lock(void *arg)
+{
+        pthread_mutex_t *l = arg;
+        pthread_mutex_unlock(l);
```

Comment:
That came due to some carry forward code to cleanup spinlocks as spinlocks are "volatile int"